client/document/index.jsx

```diff
@@ -72,6 +73,11 @@ class Document extends Component {
 			( initialReduxState
 				? `var initialReduxState = ${ jsonStringifyForHtml( initialReduxState ) };\n`
 				: '' ) +
+			( dehydratedReactQueryState
```

Comment:
For symmetry with Redux, can we call it `initialReactQueryState`? Or `initialQueryState`, now when the React branding is being removed from the library 🙂